server/cmd/utils/utils.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"math"
+	"math/rand"
+	"time"
+)
+
+const LENGTH_OF_ROOM_CODE = 4
+
+func GenerateRoomCode() int {
+	rand.Seed(time.Hour.Nanoseconds())
```

Comment:
you're only supposed to seed once, make a seeded random number generator using

```go
var seed = rand.NewSource(time.Now().UnixNano())
var rng = rand.New(seed)
```

in global scope instead